templates/frontend/pages/catalogSeriesIndex.tpl

```diff
@@ -0,0 +1,66 @@
+{**
+ * templates/frontend/pages/catalogSeries.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University Library
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @brief Display the page to view books in a series in the catalog.
+ *
+ * @uses $series Series Current series being viewed
+ * @uses $publishedMonographs array List of published monographs in this series
+ * @uses $featuredMonographIds array List of featured monograph IDs in this series
+ * @uses $newReleasesMonographs array List of new monographs in this series
+ *}
+{include file="frontend/components/header.tpl" pageTitle="plugins.block.browse.series"}
+
+<div class="page page_catalog_seriesIndex">
+
+	{* Breadcrumb *}
+	{include file="frontend/components/breadcrumbs_catalog.tpl" type="series" currentTitleKey="plugins.block.browse.series"}
+
+	<h2>
+		{translate key="plugins.block.browse.series"}
+	</h2>
+
+	{* Index with series *}
+	<div class="seriesIndex">
+	<ul>
+
+		{* Series *}
+		{if $browseSeriesFactory && $browseSeriesFactory->getCount()}
+
+		{iterate from=browseSeriesFactory item=browseSeriesItem}				
```

Comment:
Double-check the indentation here and throughout the rest of this template. HTML elements and smarty tags should be indented, like:

```
<ul>
	{if ...}
		{iterate ...}
			<li>...</li>
		{/iterate}
	{/if}
</ul>
```